### DISCUSSION TITLE
Remix and React 18

### DISCUSSION THREAD

**Discussion by rafael-franca** [tw-in-js/use-twind-with#33] [Q&A]
I'm studying Remix and tried use Twind. While stead with React 17, the application executes perfectly, However, when update to React 18, the styles initially load, but at finish of loading, lose the styles.

Anyone knows how I can resolve this?

**Comment by sastan** [comment]
React v18 is still an open issue. I'm not using react currently and did not have the time to test it. I hope to work on/look into that issue _soonish_.
